components/styled/icon/src/atoms/chevron-circle-icon.tsx

```diff
@@ -22,6 +22,9 @@ const calculateIconType = (direction: IconDirection): IconTypes => {
   }
 }
 
+const buildClassesForIcon = (className: string | undefined, direction: IconDirection): string =>
+  `${className ?? ''} icon__chevron-circle--${direction}`.trimStart()
```

Comment:
Couldn't this just be inline? Not sure what the value is in it being a function?